bravado_core/spec.py

```diff
@@ -160,6 +161,7 @@ def resolver(self):
             base_uri=self.origin_url or '',
             referrer=self.spec_dict,
             handlers=self.get_ref_handlers(),
+            store=LOCAL_JSON_SCHEMA_STORE
```

Comment:
The RefResolver class from jsonschema can be initialized with `store` to prevent us from downloading the schema from json-schema.org